### DISCUSSION TITLE
EGit and Hooks

### DISCUSSION THREAD

**Discussion by LeGrosWinnie** [eclipse-platform/eclipse.platform#1667] [Q&A]
Hello,

Question asked many times but I still have issue with hooks.
I managed to get them to be executed (add git for windows path to User PATH) but I cannot read the message.
Instead of having a console or terminal output it is a small empty popup :
![image](https://github.com/user-attachments/assets/f64684e9-1cf0-4286-8909-25de1f21851a)

Any idea ?

Thanks.

**Comment by merks** [comment]
You're asking in the wrong place.  Ask here instead:

https://github.com/eclipse-egit/egit/discussions

The forum will be shutdown shortly:

https://www.eclipse.org/forums/index.php?t=thread&frm_id=48

**Comment by akurtakov** [comment]
Eclipse platform has nothing to do with git hooks. This is for https://github.com/eclipse-egit/egit . As Github doesn't allow moving between projects - I'm closing this one and you would have to open a new one there.
